internal/config/project.go

```diff
@@ -0,0 +1,21 @@
+package config
+
+type Project struct {
```

Comment:
@psihachina What's "Project" in this case? Is it the whole monorepo? If yes, I would avoid naming it as Project, as at the moment project is something that is in the projects/ directory, so either we need to rename those or come up with a name of the "thing" that happens in the ize-structured repo.

...Maybe we should rename the projects/ directory to apps/ and then a project would be logical? 

Let me know what you think.

Reply:
Project = monorepo. Also you have a created task for rename `projects_dir` to `apps_dir`. Is there any point in discussing it then?